### DISCUSSION TITLE
High training MSE

### DISCUSSION THREAD

**Discussion by psn417** [deepmodeling/deepmd-kit#195] [Q&A] [answered]
I use 170000 AIMD frames to train a model, but error (both training and testing) is high and the melting point is different from AIMD. I try to modify the number of neurons , but result is similiar. How can I make error lower?

My Input:
{
    "_comment": " model parameters",
    "use_smooth":	false,
    "sel_a":		[40],
    "sel_r":            [60],
    "rcut":		5,    
    "n_neuron":		[240, 240, 240],
    "axis_rule":	[],

    "_comment": " traing controls",
    "systems":		["../deepmd_data/"],
    "set_prefix":	"set",    
    "stop_batch":	1000000,
    "batch_size":	1,
    "start_lr":		0.002,
    "decay_steps":	5000,
    "decay_rate":	0.95,

    "start_pref_e":	0.02,
    "limit_pref_e":	1,
    "start_pref_f":	1000,
    "limit_pref_f":	1,
    "start_pref_v":	0,
    "limit_pref_v":	0,

    "seed":		1
![Figure_6](https://user-images.githubusercontent.com/36497958/75259655-50bec280-5823-11ea-9b8e-c9fff6cb2067.png)
,

    "_comment": " display and restart",
    "_comment": " frequencies counted in batch",
    "disp_file":	"lcurve.out",
    "disp_freq":	1000,
    "numb_test":	50,
    "save_freq":	1000,
    "save_ckpt":	"model.ckpt",
    "load_ckpt":	"model.ckpt",
    "disp_training":	true,
    "time_training":	true,
    "profiling":	false,
    "profiling_file":	"timeline.json",

    "_comment":		"that's all"
}

**Comment by psn417** [comment]
When I train other phases (bcc, fcc), the error is much lower and melting point is good.

**Comment by jameswind** [comment, marked as answer]
Could you check the AIMD setup? If the model works for bcc and fcc phases I
guess maybe the numerical error of DFT in this new case have some problems.
You may pick up some snapshots in the training data, do DFT calculations
with a more strict criterion (ecut, kpoint, etc.) and see the results are
consistent with you training data or not.

Best wishes,
Linfeng
